iati-activities-schema.xsd

```diff
@@ -2806,13 +2778,25 @@
     </xsd:annotation>
   </xsd:attribute>
 
-  <xsd:attribute name="vocabulary" type="xsd:string">
-    <xsd:annotation>
-      <xsd:documentation xml:lang="en">
-        A code for the vocabulary aid-type classifications. If omitted the AidType (OECD DAC) codelist is assumed. The code must be a valid value in the AidTypeVocabulary codelist.
-      </xsd:documentation>
-    </xsd:annotation>
-  </xsd:attribute>
-
+  <xsd:complexType name="aid-type-base">
```

Comment:
I think we tend to name `complexType`s using lower camel case  - e.g. in https://github.com/IATI/IATI-Schemas/pull/372/files#diff-644a12e8a6ade95722e1f3adf60b7462R148